laravel-app/public/assets/js/likeUnlike.js

```diff
@@ -0,0 +1,68 @@
+$(document).ready(function() {
+    $('[data-is-liked]').each(function() {
+        const isLiked = $(this).data('is-liked');
+        const postId = $(this).data('post-id');
+        const likeButton = $('#saveLike[data-post-id="' + postId + '"]');
+        const unlikeButton = $('#saveUnlike[data-post-id="' + postId + '"]');
+
+        if (isLiked) {
+            likeButton.hide();
+            unlikeButton.show();
+        } else {
+            likeButton.show();
+            unlikeButton.hide();
+        }
```

Comment:
Prevent using else in the if statement!

Reply:
will fix in the next PR sir, together with fixes for blade designs and ajax follow/unfollow button. Thank you